src/horus_demod.c

```diff
@@ -227,28 +238,35 @@ int main(int argc, char *argv[]) {
 	    /* Print a sample of the FFT from the freq estimator */
 
 	    Ndft = hstates->fsk->Ndft/2;
-	    for(i=0; i<Ndft; i++){
+
+	    for (i = 0; i < Ndft; i++){
 		fprintf(stderr,"%f ",(hstates->fsk->fft_est)[i]);
-		if(i<Ndft-1) fprintf(stderr,",");
+		
+                if(i<Ndft-1)
+                    fprintf(stderr, ",");
 	    }
             #else
 
             /* All zero dummy data for now */
 
  	    Ndft = 128;
-	    for(i=0; i<Ndft; i++) {
+
+	    for (i = 0; i < Ndft; i++) {
 		fprintf(stderr,"%f ", 0.0);
-		if(i<Ndft-1) fprintf(stderr,",");
+		
+                if (i < Ndft-1)
+                    fprintf(stderr, ",");
 	    }
             
             #endif
 
 	    fprintf(stderr,"]}\n");
             stats_ctr = stats_loop;
         }
+
         stats_ctr--;
 
-        if (fin == stdin || fout == stdin){
+        if (fin == stdin || fout == stdin) {
```

Comment:
@srsampson A lot of these are personal preference style changes.  We don't really have a coding convention so I'm prepared to let individual authors use their own style.  I'm not too bothered either way, but some authors may be ... so pls be careful to check with original authors before you make these sorts of changes.

Reply:
Yes, I have a bad habit of touching code as I try to read through the compressed styles. I'll terminate that activity.

Reply:
Haha :actually it's a common habit - I do it too.

At $dayjob we have some serious style-Nazi's so this issue pops up a lot there too. I'm actually moving to a more compressed style myself but am happy to let individuals do their own thing in their modules (you get to make the call on ofdm.c for instance).  

I make a weak effort to stick with the style of the source file I am in at the time :-)

One way to handle this is to use auto-formatting, e.g. clang has emacs modules that formats the C code when you save a file.  But we would need to have everyone install it.